.yamllint

```diff
@@ -0,0 +1,6 @@
+extends: relaxed
+
+rules: 
+    line-length: 
+        max: 200 
+        allow-non-breakable-inline-mappings: true
```

Comment:
Ironically, this lack of newline would not pass the linter if this were linted.

Reply:
That's what nathanlint is for! XD